### PR TITLE
AMP-96489 Add new docs for lookup table apis v3

### PR DESCRIPTION
**Pull request by rctbusk**
# Amplitude Developer Docs PR

## Description
Added docs for the new Lookup Table API V3, which is under data-api, so moved to the Data section.

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [X] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [X] My documentation follows the style guidelines of this project.
- [X] I previewed my documentation on a local server using `mkdocs serve`.
- [X] Running `mkdocs serve` didn't generate any failures.
- [X] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/sources/lookup-table.md [hunk 1]
```diff
@@ -30,27 +30,25 @@ To set up this integration, you need the following:
 
 1. In Amplitude Data, click **Catalog** and select the **Sources** tab.
 2. In the Lookup Tables section, click **CSV**.
-3. Upload the CSV then click **Next**.
-4. Map your event property.
+3. Select column in CSV to map property to.
```
Comment (shibrady) [inline docs/data/sources/lookup-table.md:33]: Grammar seems a bit off?

File: docs/analytics/apis/lookup-tables-api.md [hunk 2]
```diff
@@ -3,8 +3,11 @@ title: Lookup Table API
 description: The Lookup Table API lets you create, get, update, and delete lookup tables to augment your data.
 ---
 
+!!!deprecated "This version of the Lookup Tables API is deprecated"
+    This API is no longer supported. Use the new [API](/data/sources/lookup-table).
```
[the remark this answers is not in the document]
Comment (rctbusk) [reply docs/analytics/apis/lookup-tables-api.md:7]: Yep, will fix :)

File: docs/data/sources/lookup-table.md (hunk 1 above)
Comment (rctbusk) [reply docs/data/sources/lookup-table.md]: Tried fixing a bit

File: docs/analytics/apis/lookup-tables-api.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rctbusk) [reply docs/analytics/apis/lookup-tables-api.md]: Fixed

File: docs/data/apis/lookup-tables-api.md [hunk 3]
```diff
@@ -0,0 +1,500 @@
...
+## Considerations
+
+The CSV file must follow the following requirements:
+
+- The max file size is 100 MB and the file can't have more than 1,000,000 rows.
+- The first row must contain column names/headers.
+- The first column must correspond to the mapping property value and must contain *unique* values. Lookup Tables search for exact matches, and are *case-sensitive*.
+- Columns are separated by commas.
+- Rows are separated by line breaks.
+- If a field value contains commas or quotes, it should be wrapped within double quotation marks. The first double quote signifies the beginning of the column data, and the last double quote marks the end. If the value contains a string with double quotes, these are replaced by two double quotes `""`.
```
Comment (shibrady) [inline docs/data/apis/lookup-tables-api.md:32]: Verbiage is inconsistent - let's just call these double quotes?

File: docs/data/apis/lookup-tables-api.md [hunk 4]
```diff
@@ -0,0 +1,500 @@
+---
+title: Lookup Table API
```
Comment (shibrady) [inline docs/data/apis/lookup-tables-api.md:2]: Casing seems inconsistent throughout the doc when it comes to lookup tables - when should and shouldn't "Lookup Table" be capitalized?

e.g. Header sections don't seem like they're title cased until Lookup Tables are involved

You can "Delete a Lookup Table", but "To create a lookup property, create a lookup table", etc.

File: docs/data/sources/lookup-table.md [hunk 5]
```diff
@@ -30,27 +30,25 @@ To set up this integration, you need the following:
 
 1. In Amplitude Data, click **Catalog** and select the **Sources** tab.
 2. In the Lookup Tables section, click **CSV**.
-3. Upload the CSV then click **Next**.
-4. Map your event property.
+3. Select column in CSV as the key column.
```
Comment (shibrady) [inline docs/data/sources/lookup-table.md:33]: Maybe "Select a column in the CSV to be the key column"?

**Review by shibrady — APPROVED** [review]

File: docs/data/sources/lookup-table.md [hunk 6]
```diff
@@ -24,33 +24,31 @@ To set up this integration, you need the following:
     - The first column must correspond to the mapping property value and must contain *unique* values. Lookup Tables search for exact matches, and are *case-sensitive*.
     - Columns must be separated by commas.
     - Rows must be separated by line breaks.
-    - If a field value contains commas or quotes, it should be wrapped within double quotation marks. The first double quote signifies the beginning of the column data, and the last double quote marks the end. If the value contains a string with double quotes, these are replaced by two double quotes `""`.
+    - If a field value contains commas or quotes, it should be wrapped within double quotation marks. The first double quote signifies the beginning of the column data, and the last double quote marks the end. If the value contains a string with double quotes, these are replaced by double quotes.
```
Comment (markzegarelli) [inline docs/data/sources/lookup-table.md:27]:
```suggestion
    - If a field value contains commas or quotes, wrap the contents with double quotation marks (`"`). The first double quote signifies the beginning of the column data, and the last double quote marks the end. If the value contains a string with double quotes, these are replaced by double quotes.
```
@rctbusk I'm not clear on the last sentence here. It seems like we should tell users to replace double-quotes within a field with single quotes, so they don't interfere with the double-quote wrapper. Is this accurate?

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: docs/data/sources/lookup-table.md (hunk 6 above)
Comment (rctbusk) [reply docs/data/sources/lookup-table.md:27]: yep! that makes sense.

File: docs/data/sources/lookup-table.md [hunk 7]
```diff
@@ -24,33 +24,31 @@ To set up this integration, you need the following:
     - The first column must correspond to the mapping property value and must contain *unique* values. Lookup Tables search for exact matches, and are *case-sensitive*.
     - Columns must be separated by commas.
     - Rows must be separated by line breaks.
-    - If a field value contains commas or quotes, it should be wrapped within double quotation marks. The first double quote signifies the beginning of the column data, and the last double quote marks the end. If the value contains a string with double quotes, these are replaced by two double quotes `""`.
+    - If a field value contains commas or quotes, wrap the contents with double quotation marks (`"`). The first double quote signifies the beginning of the column data, and the last double quote marks the end. If the value contains a string with double quotes, these are replaced by double quotes.
```
Comment (markzegarelli) [inline docs/data/sources/lookup-table.md:27]:
```suggestion
    - If a field value contains commas or quotes, wrap the contents with double quotation marks (`"`). The first double quote signifies the beginning of the column data, and the last double quote marks the end. If a string contains double quotes, replace them with single quotes.
```

**Pull request merged**